pymemcache/client/base.py

```diff
@@ -1009,6 +1009,9 @@ class PooledClient(object):
     in the pool. Your serde object must therefore be thread-safe.
     """
 
+    # default class to use for creating clients
```

Comment:
I _think_ we want to use Sphinx's `#:` comment syntax so this shows up in the generated API documentation. Also, `Client` is the default class, but the attribute holds the class that's actually going to be used.

```suggestion
    #: :class:`Client` class used to create new clients
```